tests/utils_tests/test_http.py

```diff
@@ -218,15 +218,16 @@ def test_unquote_plus(self):
 
 class IsSameDomainTests(unittest.TestCase):
     def test_good(self):
-        for pair in (
-            ('example.com', 'example.com'),
-            ('example.com', '.example.com'),
-            ('foo.example.com', '.example.com'),
-            ('example.com:8888', 'example.com:8888'),
-            ('example.com:8888', '.example.com:8888'),
-            ('foo.example.com:8888', '.example.com:8888'),
+        for pair, expected in (
+            (('example.com', 'example.com'), True),
+            (('example.com', '.example.com'), True),
+            (('foo.example.com', '.example.com'), True),
+            (('example.com:8888', 'example.com:8888'), True),
+            (('example.com:8888', '.example.com:8888'), True),
+            (('foo.example.com:8888', '.example.com:8888'), True),
+            (('foo.example.com:8888', None), False),
         ):
```

Comment:
This would be a good opportunity to use `subTest`.

Reply:
@charettes thank you. I changed it and use `subTest`.